Dockerfile

```diff
@@ -39,11 +34,13 @@ WORKDIR /home/ethsec
 ENV HOME="/home/ethsec"
 ENV PATH="${PATH}:${HOME}/.local/bin"
 
-# Select the latest version of solc as the default:
-RUN solc-select --list | tail -n1 | xargs solc-select
-
 RUN mv examples etheno-examples
 
+# Install all and select the latest version of solc as the default
+# SOLC_VERSION is defined to a valid version to avoid a warning message on the output
+RUN pip3 --no-cache-dir install solc-select
+RUN solc-select install all && SOLC_VERSION=0.8.0 solc-select versions | head -n1 | xargs solc-select use
```

Comment:
Maybe this is unnecessary. It will make the image considerably heavier to download.